addons/libs/images.lua

```diff
@@ -387,17 +398,25 @@ windower.register_event('mouse', function(type, x, y, delta, blocked)
     if type == 0 then
         if dragged then
             dragged.image:pos(x - dragged.x, y - dragged.y)
+            call_events(dragged.image, 'drag')
+            dragged.image:last_pos(dragged.image:pos())
```

Comment:
This needs to be called after the call_events, not before, in order for the event to retain access to the previous last_pos. Update this after the event so that the next drag event has an updated last_pos.